pkg/spireentry/reconciler.go

```diff
@@ -97,15 +99,15 @@ func (r *entryReconciler) reconcile(ctx context.Context) {
 	}
 
 	// Load and add entry state for ClusterStaticEntries
-	clusterStaticEntries, err := r.listClusterStaticEntries(ctx)
+	clusterStaticEntries, err := r.listClusterStaticEntries(ctx, r.config.ClassName, r.config.MissingClassName)
```

Comment:
nit: the reconciler list methods already have access to the config; no need to pass these config values.

Reply:
Wasn't sure how to get access to it. There an example I missed? Happy to remove it.

Reply:
oh. because they are methods of r already. gotcha. Will fix.

Reply:
Done